tests/pom.xml

```diff
@@ -104,7 +104,7 @@
         </dependency>
         <dependency>
             <groupId>org.mockito</groupId>
-            <artifactId>mockito-inline</artifactId>
+            <artifactId>mockito-core</artifactId>
             <version>${mockito.version}</version>
```

Comment:
The version should be specified by the `dependencyManagement` section of the parent pom, so it doesn't need to be specified here.
```suggestion
```